ocw/lib/openstack.py

```diff
@@ -23,14 +24,17 @@ def __new__(cls, vault_namespace: str):
     def client(self) -> None:
         if self.__client is None:
             self.__client = openstack.connect(
+                debug=DEBUG,
                 insecure=True,  # Trust the certificate
                 auth_url=self.get_data('auth_url'),
                 project_name=self.get_data('project_name'),
                 username=self.get_data('username'),
                 password=self.get_data('password'),
                 region_name=self.get_data('region_name'),
                 user_domain_name=self.get_data('user_domain_name'),
-                project_domain_name=self.get_data('project_domain_id'),
+                project_id=self.get_data('project_id'),
+                load_envvars=False,  # Avoid reading OS_* environment variables
```

Comment:
can elaborate why both needs to be avoided ? ( I am not against just feel that there is some story behind which I want to know ... )

Reply:
The original code worked on my machine because I had those environment variables turned on, but I was assigning `project_domain_id` to `project_domain_name` and our Openstack instance was getting info from a SUMA project for some reason (maybe because it's default).

Reply:
ok